connectors/spark/src/main/scala/io/unitycatalog/spark/auth/GenericCredentialProvider.java

```diff
@@ -17,9 +17,6 @@
 import org.sparkproject.guava.cache.CacheBuilder;
 
 public abstract class GenericCredentialProvider {
-  // The remaining time before expiration, used to trigger renewal in advance.
-  private static final long DEFAULT_RENEWAL_LEAD_TIME_MILLIS = 30 * 1000;
```

Comment:
This can be removed, because we just introduced a hadoop configuration key, so that we can also set it the test suites. That will be more convinient to do the integration testing, since for the IT, we don't have API to explicitly use the `GenericCredentialsProvider`'s constructor to pass `renewalLeadTime` to construct --- All the providers are initialized via the java reflection. 

The only approach is using the `hadoop` configuration setting.